pages/docs/fundamentals/fundamentals-roadmap.md

```diff
@@ -14,20 +14,15 @@ If you are looking for features introduced already in the past, have a look at o
 
 ## In active development
 
-- [Waveform relaxation for multi-rate coupling and higher-order time stepping](https://github.com/precice/precice/projects/7)
-- [Partition-of-Unity RBF data mapping for very large problems](https://github.com/precice/precice/issues/1273)
 - [GPU support for data mapping methods](https://github.com/precice/precice/issues/1484)
 - [Coupling with Functional Mock-Up Units](https://github.com/precice/fmi-runner)
 - [Adaptive and flexible macro-micro coupling software](https://github.com/precice/micro-manager)
```

Comment:
```suggestion
```

Reply:
Isn't this still in development though?

Reply:
Too generic a topic, and the development has progressed substantially now. There are niche topics available, but then they need to be specified clearly.